src/goto-instrument/nondet_static.cpp

```diff
@@ -18,6 +18,7 @@ Date: November 2011
 #include <util/std_expr.h>
 #include <util/cprover_prefix.h>
 #include <util/prefix.h>
+#include <util/type.h>
```

Comment:
That include is unnecessary (it's dragged in via `std_expr.h` already).

Reply:
@tautschnig  There is some discussion that we shouldn't depend on transitive `#include`s. Do you think it definitely needs to go?

Reply:
Oh, I was not aware of that discussion. I was just very surprised to see this change.